src/Topshelf.Unity/Topshelf.Unity.nuspec

```diff
@@ -13,7 +13,7 @@
     <copyright>© 2014-2016 Alexandr Nikitin. All rights reserved.</copyright>
     <tags>Topshelf Unity</tags>
     <dependencies>
-      <dependency id="Unity" version="[4.0.0,5.0.0)" />
+      <dependency id="Unity.Abstractions" version="2.1.1" />
```

Comment:
Can we relax the version constraint to something like [2.0.0, 3.0.0) WDYT?

Reply:
I am not sure about reference to Unity.Abstractions in nuspec.
Maybe it's better if NuGet package depends on main Unity 5.1.x.

Reply:
I think it's fine since Topshelf.Unity depends on Abstractions only, ie `IUnityContainer` interface. It's up to a client to bring the Unity implementation.